pyirf/utils.py

```diff
@@ -88,6 +90,34 @@ def calculate_source_fov_offset(events, prefix="true"):
     return theta.to(u.deg)
 
 
+def calculate_source_fov_position_angle(events, prefix="true"):
+    """Calculate position_angle of true positions relative to pointing positions.
+
+    Parameters
+    ----------
+    events : astropy.QTable
+        Astropy Table object containing the reconstructed events information.
+
+    prefix: str
+        Column prefix for az / alt, can be used to calculate reco or true
+        source fov position_angle.
+
+    Returns
+    -------
+    phi: astropy.units.Quantity
+        Position angle of the true positions relative to the pointing positions
+        in the sky.
+    """
+    phi = position_angle(
+        events["pointing_az"],
```

Comment:
I am not sure this definition matches the defintion in GADF given here:
https://gamma-astro-data-formats.readthedocs.io/en/v0.3/general/coordinates.html#field-of-view

Reply:
Please add tests for that